pymedphys/_trf/decode/header.py

```diff
@@ -33,7 +33,7 @@ def determine_header_length(trf_contents):
 def decode_header(trf_header_contents):
     match = re.match(
         br"[\x00-\x19]"  # start bit
-        br"(\d\d/\d\d/\d\d \d\d:\d\d:\d\d Z)"  # date
+        br"(\d\d[/-]\d\d[/-]\d\d \d\d:\d\d:\d\d Z)"  # date
```

Comment:
what other date formats might be expected?
Dutch timestamp format is presumably not really just Dutch, but might include other parts of Western Europe.  Which then raises the question regarding what other region formats might be expected.  Might Eastern Europe use a different date format?  Different places in Asia (Japan, Korea, China)?

Is anyone concerned about rolling over the century mark (previously known as the Y2K problem)?

Further down, the bytes are being decoded as ascii.
but the regex that got parsed includes up to \xFF, while ascii is only up to \xEF or maybe \xF0.
so it is really ascii or is it Latin1 (or iso-8859, or something similar)?

Reply:
Fair point about decoding with ASCII, I should possibily be using something else for sure. Would be helpful to have someone create a field with some very weird characters in it and then get the resulting TRF file.

Regarding most of the other points, I am far more comfortable only actually making changes to the decoding logic if a file shows up that isn't properly decoded. And then make the changes to appropriately decode said file with a change. Not so keen on trying to predict what possible formats of TRF are out there.

Reply:
>Not so keen on trying to predict what possible formats of TRF are out there.

Without the official documentation, that's a very fair approach.